protractor-docker/docker.md

```diff
@@ -0,0 +1,151 @@
+Running Protractor Tests on  Docker
+========================================
+
+This is a simple tutorial that shows how to use docker to run Protractor tests. We would setup a Selenium Grid and run the tests.
+
+Prerequisites
+-------------
+Docker needs to be installed on your machine. One can download it from [Docker's website](https://www.docker.com) and follow the [documentation](https://docs.docker.com/) accordingly.
+It is assumed that , one knows the basics of Docker for this tutorial.
+
+Setup
+-----------
+To ensure Docker is installed sucessfully , type :
+``` shell
+docker -v
+```
+and one  would see a similar output , depending on the version of docker installed :
+``` shell
+Docker version 1.12.0-rc4, build e4a0dbc, experimental
+``` 
+
+Step 0 - Download the docker images for Selenium Hub and Selenium Node
+-----------------------------------------------------------------------------
+
+``` shell
+docker pull selenium/hub:latest
+docker pull selenium/node-chrome:latest
+```
+One could pull 'node-firefox' if they want to work with firefox node. 
+For more information about the different images one can work with , please look at [Docker Selenium Images List](https://github.com/SeleniumHQ/docker-selenium/blob/master/README.md)
+
+
+Step 1 - Starting the Selenium Grid
+-----------------------------------------------------------------------------
+``` shell
+docker run -d -p 4444:4444 --name selenium-hub selenium/hub:latest
+```
+
+One can change the tag from 'latest' to '2.53.0' or any other version as they see fit.
+
+Step 2 - Starting the Selenium Nodes
+-----------------------------------------------------------------------------
+``` shell
+docker run -d --link selenium-hub:hub selenium/node-chrome:latest
+docker run -d --link selenium-hub:hub selenium/node-firefox:latest
```

Comment:
If this example is just showing the latest `node-chrome`, let's take out the `node-firefox`